Description: This patch changes the "FlowLayout::expandingDirections" function according to the new syntax used in Qt 5.15.

<https://doc.qt.io/qt-5.15/qtwidgets-layouts-flowlayout-example.html>

diff --unified --recursive --text olive-0.1.2-orig/ui/flowlayout.cpp olive-0.1.2-new/ui/flowlayout.cpp
--- olive-0.1.2-orig/ui/flowlayout.cpp	2019-11-11 03:05:02.000000000 -0300
+++ olive-0.1.2-new/ui/flowlayout.cpp	2020-06-23 18:29:57.338203268 -0300
@@ -114,7 +114,7 @@
 
 Qt::Orientations FlowLayout::expandingDirections() const
 {
-    return 0;
+    return { };
 }
 
 bool FlowLayout::hasHeightForWidth() const
